File: level 30/homework/hw1.py
# 1) შექმენით ლისტი სადაც გექნებათ 1-10 ჩათვლით რიცხვები და დაპრინტეთ მხოლოდ ლუწი რიცხვები

evens_list=[1,2,3,4,5,6,7,8,9,10]
for i in range(len(evens_list)):
    if evens_list[i]%2==0:
        print(evens_list[i])

File: level 105/homework/hw1.js
// 1)დაწერე ფუნქცია, რომელიც იღებს არგუმენტად რიცხვების სიას და ახალ სიაში ამატებს მხოლოდ ლუწ რიცხვებს. შემდეგ იპოვეთ ამ სიაში მყოფი რიცხვების ჯამი
function func(numbersList){
    let evensList=[]
    let i=0
    while(i<numbersList.length){
        if(numbersList[i]%2==0){
            evensList.push(numbersList[i])
        }
        i++
    }
    let sum=0
    for(i=0;i<evensList.length;i++){
        sum+=evensList[i]
    }
    return sum
}

console.log(func([34,45,2,564,99,5]))


